### PR TITLE
Create 2019-07-10-prestashop-1-7-6-0-available.md

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: news/_posts/releases/2019-07-10-prestashop-1-7-6-0-available.md [hunk 1]
```diff
@@ -0,0 +1,192 @@
+---
+layout: post
+title:  "PrestaShop 1.7.6.0 is available!"
+subtitle: "You waited for it, here it is!"
+date:   2019-07-10 18:00:00
```
Comment (matks) [inline news/_posts/releases/2019-07-10-prestashop-1-7-6-0-available.md]: Do you plan to publish it today ?

**Review by Progi1984 — APPROVED** [review]

File: news/_posts/releases/2019-07-10-prestashop-1-7-6-0-available.md (hunk 1 above)
Comment (ghost) [reply news/_posts/releases/2019-07-10-prestashop-1-7-6-0-available.md]: Well, the plan was to make the 176 available tonight on autoupgrade and the dotcom, but to wait for tomorrow to send the emails to the 2M PS newsletter subscribers + publish the blogposts in 8 languages on the dotcom.

If we make it available tonight we have to publish the release note so it's official and note alone by itself in the wild. I'll check with Coline to be sure.

Comment (eternoendless) [reply news/_posts/releases/2019-07-10-prestashop-1-7-6-0-available.md]: Be aware that if you publish an article in the future it won't show up until you rebuild jekyll. I changed it to 17:00 to be sure

**Review by eternoendless — APPROVED** [review]

**Comment by eternoendless** [comment]
Thank you @samuel-pires

**Comment by ghost** [comment]
Thanks for the review and modifications @eternoendless, looks great!

File: news/_posts/releases/2019-07-10-prestashop-1-7-6-0-available.md [hunk 2]
```diff
@@ -0,0 +1,194 @@
...
+
+- Improved Catalog mode: for showcase websites, you can now choose to display or to hide product prices.
+- Improved currency management subsystem. This is the basis to allow more customization in the next versions, adding custom currencies for example.
+- Improved SEO for products with combinations.
+
+**Back office**
+- Manage and preview your transactional emails directly from a new back-office page: Email Theme.
+- In order to gather all prices in the same place, a “Price (tax incl.)” column has been added to the product listing. In one glance, it is now possible to get both tax included and excluded prices for each product.
+- New helper cards have been added on important pages to improve the software onboarding: Categories, Customers, Pages, Employees.
+- Dynamic preview of search engine results have been added to more pages: Product page, Categories, CMS.
```
Comment (matks) [inline news/_posts/releases/2019-07-10-prestashop-1-7-6-0-available.md:72]: `have` => `has` ?

File: news/_posts/releases/2019-07-10-prestashop-1-7-6-0-available.md [hunk 3]
```diff
@@ -0,0 +1,194 @@
...
+- New CLDR standard implementation: improved localization of prices and amounts, will allow creating custom currencies in future versions
+- Support for multiple grids on the same page
+- Performance:
+  - Improved employee authorization process
+  - Cache introduced when loading modules
+  - The number of SQL requests to get the employee authorizations has been significantly reduced
+- Web services:
+  - Product customization is now reachable
+  - Product combination information in Pack content is now available
+
```
Comment (matks) [inline news/_posts/releases/2019-07-10-prestashop-1-7-6-0-available.md:119]: Lot of blank lines ?

File: news/_posts/releases/2019-07-10-prestashop-1-7-6-0-available.md [hunk 4]
```diff
@@ -0,0 +1,194 @@
...
+  - actionBuildLayoutVariables
+  - actionGetMailLayoutTransformations
+  
+See all hooks in the [DevDocs](https://devdocs.prestashop.com/1.7/modules/concepts/hooks/list-of-hooks/)
+
+
+### Improved hooks
+
+- actionProductAdd now receives id_product_old when duplicating a product
+
```
Comment (matks) [inline news/_posts/releases/2019-07-10-prestashop-1-7-6-0-available.md:141]: Lot of blank lines ?

File: news/_posts/releases/2019-07-10-prestashop-1-7-6-0-available.md [hunk 5]
```diff
@@ -0,0 +1,194 @@
...
+### Breaking or risky changes
+
+- The new CLDR implementation replaces the old one. Some breaking changes were needed:
+  - The Currency ObjectModel is now multilingual
+  - The IcanBoogie/CLDR library has been removed
+  - Calls to Tools::getCldr() throw a PrestaShopException
+  - The namespace “PrestaShop\PrestaShop\Core\Cldr” has been removed
+  - The CLDR files in the “/translations” directory have been removed
+- Import:
+  - You no longer can just add categories to current categories with import to a force ID. You will have to set them all and the first will become the default_category.)
```
Comment (matks) [inline news/_posts/releases/2019-07-10-prestashop-1-7-6-0-available.md:153]: `You no longer can` => `You can no longer`
And there is a `)` at the end of the line, but no `(` anywhere

File: news/_posts/releases/2019-07-10-prestashop-1-7-6-0-available.md [hunk 6]
```diff
@@ -0,0 +1,194 @@
...
+- Replaced library:
+  - PhpExcel (no longer supported) has been replaced by PHPSpreadsheet)
```
Comment (matks) [inline news/_posts/releases/2019-07-10-prestashop-1-7-6-0-available.md:155]: Ending `)` but no opening `(`

**Pull request merged**